### ISSUE TITLE
Support generating random strings from regex payload generator

### ISSUE THREAD

**Issue by AndiDog** [zaproxy/zaproxy#1886]
For example with `[0-9]+`, the generator will produce 0, 1, 2, ..., 00, 01 and so on. For attack purposes, it'd be nice to have a checkbox "Random generation" which causes the selection of e.g. `[a-z]` expressions to be randomized. Also the length could be randomized (depends on issue #1885), e.g. `[a-f]{3,5}` with randomization could output "a0fd", "f29c", "ffa", "b6c3", ...

**Comment by thc202** [comment]
Closing, this is already possible (somewhat, it might end up with duplicated payloads).
